resources/sass/core/_code.scss

```diff
@@ -3,6 +3,25 @@ a code {
   color: $primary;
 }
 
+kbd {
+  display: inline;
+  border-radius: 4px;
```

Comment:
to chyba też nie jest potrzebne. zostanie wzięty domyślny `border-radius` z bootstrapa